todo_app/app.py

```diff
@@ -7,9 +9,16 @@
 
 
 @app.route('/')
-def index():
-    return 'Hello World!'
+def get_items():
+    items = todo_app.data.session_items.get_items()
+    return render_template("index.html", items=items)
 
+@app.route('/add_item', methods=['POST', 'GET'])
+def add_new_item():
+    title = request.form['title']
+    todo_app.data.session_items.add_item(title)
+    return redirect(os.getenv("HOSTNAME_GITPOD"))
```

Comment:
As you mentioned yourself before, the code should not be specific to running in GitPod. So this should be changed to:
```suggestion
    return redirect(os.getenv("HOSTNAME"))
```

And it will work as long as the environment is set up correctly (during dev that means `.env` needs the correct value for HOSTNAME=...)

Reply:
Done